### ISSUE TITLE
Error Cant put to work my bot :S

### ISSUE THREAD

**Issue by xRpwn** [Necrobot-Private/NecroBot#509]
### Actual Behavior:
[10:38:47] (UPDATE) Perfect! You already have the newest Version 1.0.0.54
[10:38:47] (INFO) Logging in using Google
Starting ChromeDriver 2.26.436362 (5476ec6bf7ccbada1734a0cdec7d570bb042aa30) on port 59110
Only local connections are allowed.
[10:38:51] (INFO) Exception caught, writing LogBuffer.

Excepção não processada: PokemonGo.RocketAPI.Exceptions.CaptchaException: Tipo de excepção 'PokemonGo.RocketAPI.Exceptions.CaptchaException' accionado.
   em PokemonGo.RocketAPI.Helpers.CommonRequest.ProcessCheckChallengeResponse(Client client, CheckChallengeResponse checkChallengeResponse) em c:\projects\necrobot2\FeroxRev\Helpers\CommonRequest.cs:line 199
   em PokemonGo.RocketAPI.Rpc.Player.<GetPlayer>d__4.MoveNext() em c:\projects\necrobot2\FeroxRev\Rpc\Player.cs:line 80
--- Fim do rastreio da pilha da localização anterior em que a excepção foi emitida ---
   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   em PoGo.NecroBot.Logic.Inventory.<GetPlayerData>d__27.MoveNext() em c:\projects\necrobot2\PoGo.NecroBot.Logic\Inventory.cs:line 353
--- Fim do rastreio da pilha da localização anterior em que a excepção foi emitida ---
   em System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_1(Object state)
   em System.Threading.QueueUserWorkItemCallback.WaitCallback_Context(Object state)
   em System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   em System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   em System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   em System.Threading.ThreadPoolWorkQueue.Dispatch()
   em System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()

### Your config.json:
```
{
  "ConsoleConfig": {
    "TranslationLanguageCode": "pt",
    "StartupWelcomeDelay": false,
    "AmountOfPokemonToDisplayOnStart": 2,
    "DetailedCountsBeforeRecycling": true
  },
  "UpdateConfig": {
    "SchemaVersion": 6,
    "CheckForUpdates": true,
    "AutoUpdate": true
  },
  "WebsocketsConfig": {
    "UseWebsocket": true,
    "WebSocketPort": 9515
  },
  "LocationConfig": {
    "DisableHumanWalking": false,
    "StartFromLastPosition": true,
    "DefaultLatitude": 38.7686316,
    "DefaultLongitude": -9.0958580000000211,
    "WalkingSpeedInKilometerPerHour": 20.0,
    "UseWalkingSpeedVariant": false,
    "WalkingSpeedVariant": 1.2,
    "ShowVariantWalking": false,
    "RandomlyPauseAtStops": true,
    "MaxSpawnLocationOffset": 10,
    "MaxTravelDistanceInMeters": 1000
  },
  "TelegramConfig": {
    "UseTelegramAPI": false,
    "TelegramAPIKey": null,
    "TelegramPassword": null
  },
  "GPXConfig": {
    "UseGpxPathing": false,
    "GpxFile": "GPXPath.GPX"
  },
  "SnipeConfig": {
    "UseSnipeLocationServer": false,
    "SnipeLocationServer": "localhost",
    "SnipeLocationServerPort": 16969,
    "GetSniperInfoFromPokezz": false,
    "GetOnlyVerifiedSniperInfoFromPokezz": true,
    "GetSniperInfoFromPokeSnipers": false,
    "GetSniperInfoFromPokeWatchers": false,
    "GetSniperInfoFromSkiplagged": false,
    "MinPokeballsToSnipe": 20,
    "MinPokeballsWhileSnipe": 0,
    "MinDelayBetweenSnipes": 60000,
    "SnipingScanOffset": 0.005,
    "SnipeAtPokestops": false,
    "SnipeIgnoreUnknownIv": false,
    "UseTransferIvForSnipe": false,
    "SnipePokemonNotInPokedex": false,
    "UseSnipeLimit": true,
    "SnipeRestSeconds": 600,
    "SnipeCountLimit": 39,
    "ActivateMSniper": true,
    "MinIVForAutoSnipe": 100,
    "AutosnipeVerifiedOnly": false,
    "SnipePauseOnOutOfBallTime": 5
  },
  "HumanWalkSnipeConfig": {
    "Enable": true,
    "DisplayPokemonList": true,
    "MaxDistance": 1500.0,
    "MaxEstimateTime": 900.0,
    "CatchEmAllMinBalls": 50,
    "TryCatchEmAll": true,
    "CatchPokemonWhileWalking": true,
    "SpinWhileWalking": true,
    "AlwaysWalkback": false,
    "SnipingScanOffset": 0.025,
    "WalkbackDistanceLimit": 300.0,
    "IncludeDefaultLocation": true,
    "UseSnipePokemonList": true,
    "MaxSpeedUpSpeed": 60.0,
    "AllowSpeedUp": true,
    "DelayTimeAtDestination": 10000,
    "UsePokeRadar": true,
    "UseSkiplagged": true,
    "UsePokecrew": true,
    "UsePokesnipers": true,
    "UsePokeZZ": true,
    "UsePokeWatcher": true,
    "UseFastPokemap": true,
    "UsePogoLocationFeeder": false,
    "AllowTransferWhileWalking": false
  },
  "DataSharingConfig": {
    "EnableSyncData": true,
    "AutoSnipe": false,
    "DataServiceIdentification": "",
    "DataRecieverURL": "ws://necrosocket.herokuapp.com/socket.io/?EIO=3&transport=websocket"
  },
  "PokeStopConfig": {
    "UsePokeStopLimit": true,
    "PokeStopLimit": 1998,
    "PokeStopLimitMinutes": 1470
  },
  "GymConfig": {
    "Enable": false,
    "PrioritizeGymOverPokestop": true,
    "MaxDistance": 500.0,
    "DefaultTeam": "Yellow",
    "MaxCPToDeploy": 1800,
    "MaxLevelToDeploy": 16,
    "VisitTimeout": 60,
    "UseRandomPokemon": false,
    "NumberOfTopPokemonToBeExcluded": 10,
    "CollectCoinAfterDeployed": 1,
    "MaxGymLevelToAttack": 3,
    "MaxDefendersToAttack": 3
  },
  "PokemonConfig": {
    "CatchPokemon": true,
    "DelayBetweenPokemonCatch": 2000,
    "UseCatchLimit": true,
    "CatchPokemonLimit": 998,
    "CatchPokemonLimitMinutes": 1470,
    "UseIncenseConstantly": false,
    "UseEggIncubators": true,
    "UseLimitedEggIncubators": true,
    "UseLuckyEggConstantly": false,
    "UseLuckyEggsMinPokemonAmount": 30,
    "UseLuckyEggsWhileEvolving": false,
    "UseBerriesMinCp": 1000,
    "UseBerriesMinIv": 90.0,
    "UseBerriesBelowCatchProbability": 0.2,
    "UseBerriesOperator": "or",
    "MaxBerriesToUsePerPokemon": 30,
    "TransferWeakPokemon": false,
    "TransferDuplicatePokemon": true,
    "TransferDuplicatePokemonOnCapture": true,
    "RenamePokemon": false,
    "RenameOnlyAboveIv": false,
    "RenameTemplate": "{1}_{0}",
    "FavoriteMinIvPercentage": 95.0,
    "AutoFavoritePokemon": false,
    "MaxPokeballsPerPokemon": 6,
    "UseGreatBallAboveCp": 1000,
    "UseUltraBallAboveCp": 1250,
    "UseMasterBallAboveCp": 1500,
    "UseGreatBallAboveIv": 85.0,
    "UseUltraBallAboveIv": 95.0,
    "UseGreatBallBelowCatchProbability": 0.2,
    "UseUltraBallBelowCatchProbability": 0.1,
    "UseMasterBallBelowCatchProbability": 0.05,
    "AutomaticallyLevelUpPokemon": false,
    "OnlyUpgradeFavorites": true,
    "UseLevelUpList": true,
    "AmountOfTimesToUpgradeLoop": 5,
    "GetMinStarDustForLevelUp": 5000,
    "LevelUpByCPorIv": "iv",
    "UpgradePokemonCpMinimum": 1000.0,
    "UpgradePokemonIvMinimum": 95.0,
    "UpgradePokemonMinimumStatsOperator": "and",
    "EvolveAboveIvValue": 95.0,
    "EvolveAllPokemonAboveIv": false,
    "EvolveAllPokemonWithEnoughCandy": true,
    "EvolveKeptPokemonsAtStorageUsagePercentage": 90.0,
    "EvolveKeptPokemonIfBagHasOverThisManyPokemon": 120,
    "KeepPokemonsThatCanEvolve": false,
    "KeepMinCp": 1250,
    "KeepMinIvPercentage": 90.0,
    "KeepMinLvl": 6,
    "KeepMinOperator": "or",
    "UseKeepMinLvl": false,
    "PrioritizeIvOverCp": true,
    "KeepMinDuplicatePokemon": 1,
    "UsePokemonToNotCatchFilter": true,
    "UsePokemonSniperFilterOnly": false,
    "DumpPokemonStats": false,
    "DelayBetweenPokemonUpgrade": 10000,
    "OutOfBallCatchBlockTime": 5,
    "PokeballToKeepForSnipe": 50,
    "UseBulkTransferPokemon": true,
    "BulkTransferStogareBuffer": 10,
    "BulkTransferSize": 100
  },
  "RecycleConfig": {
    "VerboseRecycling": true,
    "RecycleInventoryAtUsagePercentage": 90.0,
    "RandomizeRecycle": false,
    "RandomRecycleValue": 5,
    "TotalAmountOfPokeballsToKeep": 120,
    "TotalAmountOfPotionsToKeep": 80,
    "TotalAmountOfRevivesToKeep": 60,
    "TotalAmountOfBerriesToKeep": 50,
    "MaxPokeballsToKeep": 50
  },
  "CustomCatchConfig": {
    "EnableHumanizedThrows": true,
    "EnableMissedThrows": true,
    "ThrowMissPercentage": 25,
    "NiceThrowChance": 40,
    "GreatThrowChance": 30,
    "ExcellentThrowChance": 10,
    "CurveThrowChance": 90,
    "ForceGreatThrowOverIv": 90.0,
    "ForceExcellentThrowOverIv": 95.0,
    "ForceGreatThrowOverCp": 1000,
    "ForceExcellentThrowOverCp": 1500,
    "UseTransferFilterToCatch": true
  },
  "PlayerConfig": {
    "DelayBetweenPlayerActions": 4000,
    "EvolveActionDelay": 20000,
    "TransferActionDelay": 5000,
    "RecycleActionDelay": 1000,
    "RenamePokemonActionDelay": 2000,
    "UseNearActionRandom": true
  },
  "SoftBanConfig": {
    "FastSoftBanBypass": false,
    "ByPassSpinCount": 1
  },
  "GoogleWalkConfig": {
    "UseGoogleWalk": true,
    "DefaultStepLength": 1.3,
    "GoogleHeuristic": "walking",
    "GoogleAPIKey": null,
    "Cache": true,
    "GoogleElevationAPIKey": null
  },
  "YoursWalkConfig": {
    "UseYoursWalk": false,
    "YoursWalkHeuristic": "bicycle"
  },
  "MapzenWalkConfig": {
    "UseMapzenWalk": false,
    "MapzenTurnByTurnApiKey": null,
    "MapzenWalkHeuristic": "bicycle",
    "MapzenElevationApiKey": null
  },
  "ItemRecycleFilter": [
    {
      "Key": "itemUnknown",
      "Value": 0
    },
    {
      "Key": "itemLuckyEgg",
      "Value": 200
    },
    {
      "Key": "itemIncenseOrdinary",
      "Value": 100
    },
    {
      "Key": "itemIncenseSpicy",
      "Value": 100
    },
    {
      "Key": "itemIncenseCool",
      "Value": 100
    },
    {
      "Key": "itemIncenseFloral",
      "Value": 100
    },
    {
      "Key": "itemTroyDisk",
      "Value": 100
    },
    {
      "Key": "itemXAttack",
      "Value": 100
    },
    {
      "Key": "itemXDefense",
      "Value": 100
    },
    {
      "Key": "itemXMiracle",
      "Value": 100
    },
    {
      "Key": "itemSpecialCamera",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasicUnlimited",
      "Value": 100
    },
    {
      "Key": "itemIncubatorBasic",
      "Value": 100
    },
    {
      "Key": "itemPokemonStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemItemStorageUpgrade",
      "Value": 100
    },
    {
      "Key": "itemPokeBall",
      "Value": 50
    }
  ],
  "PokemonsNotToTransfer": [
    "venusaur",
    "charizard",
    "blastoise",
    "clefable",
    "muk",
    "chansey",
    "gyarados",
    "lapras",
    "ditto",
    "vaporeon",
    "jolteon",
    "flareon",
    "snorlax",
    "articuno",
    "zapdos",
    "moltres",
    "dragonite",
    "mewtwo",
    "mew"
  ],
  "PokemonsToEvolve": [
    "caterpie",
    "weedle",
    "pidgey",
    "rattata"
  ],
  "PokemonsToLevelUp": [
    "venusaur",
    "charizard",
    "blastoise",
    "clefable",
    "muk",
    "chansey",
    "gyarados",
    "ditto",
    "snorlax",
    "articuno",
    "zapdos",
    "moltres",
    "dragonite",
    "mewtwo",
    "mew"
  ],
  "PokemonsToIgnore": [
    "tauros",
    "kangaskhan",
    "mrMime",
    "farfetchd"
  ],
  "CaptchaConfig": {
    "AllowManualCaptchaResolve": true,
    "ManualCaptchaTimeout": 120,
    "PlaySoundOnCaptcha": true,
    "DisplayOnTop": true,
    "Enable2Captcha": false,
    "EnableAntiCaptcha": false,
    "AntiCaptchaAPIKey": "201.33.206.229",
    "ProxyHost": "",
    "ProxyPort": 3128,
    "EnableCaptchaSolutions": false,
    "CaptchaSolutionAPIKey": "",
    "CaptchaSolutionsSecretKey": "",
    "AutoCaptchaTimeout": 120,
    "AutoCaptchaRetries": 3,
    "TwoCaptchaAPIKey": ""

```

Version:
Windows Version: windows 10 32x
What is affected (catch, transfer, favorite, GPX, evolve, recycle, snipe, human walk snipe, pokestops, powerup/levelup, other):

**Comment by xRpwn** [comment]
Guys i got that error and can get it to work fine. iam using NecroBot2.Console. I have put to run the NecroBot2.win and is not working to :S need some help guys

**Comment by boscorelly** [comment]
same as here : https://github.com/Necrobot-Private/NecroBot/issues/501

**Comment by xRpwn** [comment]
ok guys problem is solved whit the patch 1.0.0.56 tks guys

**Comment by xRpwn** [comment]
:D
